### PR TITLE
Add native ARMv4T, ARMv5TE and ARMv8-A support

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following:** 

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Add support for pre-compiled ARMv4R, ARMv5TE and ARMv8-A FTL binaries

**How does this PR accomplish the above?:**

Improve detection and add new criteria

**Dependencies of this PR**

This PR depends on https://github.com/pi-hole/docker-base-images/pull/31 (:heavy_check_mark:) and should be merged at the same time as https://github.com/pi-hole/FTL/pull/903 (already approved)
**What documentation changes (if any) are needed to support this PR?:**

None

### CODE REVIEW

**Comment by PromoFaux** [comment]
I think you also will need to fix up the tests for these, too.

**Comment by DL6ER** [comment]
Had to force-push once again to trigger the Travis build (it was always picking the second last commit, otherwise)

**Comment by DL6ER** [comment]
@PromoFaux Tests are all :heavy_check_mark:

**Comment by DL6ER** [comment]
https://github.com/pi-hole/FTL/pull/903 should be merged (already approved) immediately after merging this PR

**Review by PromoFaux — APPROVED** [review]

**Comment by PromoFaux** [comment]
I'll let you do the honours!

**Pull request merged**
